### DISCUSSION TITLE
Issue with unformatted error logs in vitest extension

### DISCUSSION THREAD

**Discussion by aimad-majdou** [vitest-dev/vscode#324] [Q&A]
Hello,

I'm encountering an issue with the vitest extension in Visual Studio Code where error logs for certain tests are displaying as unformatted, making it difficult to interpret the results accurately.

For example, I see logs like this:

![image](https://github.com/vitest-dev/vscode/assets/16216620/62695b5f-b9d4-4bbc-a7db-85d2e3d457fe)

Is there a configuration option that I might have missed to enable proper formatting of error logs?

Thank you for your assistance in resolving this issue.

> Additional Information:
> 
> OS: Windows 10
> VSCode Version: 1.87.2
> vitest Version: v0.4.1

**Comment by sheremet-va** [comment]
Try the pre-release version
